pyirf/irf/effective_area.py

```diff
@@ -26,10 +27,9 @@ def effective_area(n_selected, n_simulated, area):
     return (n_selected / n_simulated) * area
 
 
-def point_like_effective_area(selected_events, simulation_info, true_energy_bins):
+def effective_area_energy(selected_events, simulation_info, true_energy_bins):
```

Comment:
maybe `*_per_energy` would be more explicit?

Reply:
yes